AlphaWallet/Tokens/ViewModels/AddHideTokensViewModel.swift

```diff
@@ -19,10 +19,10 @@ private enum AddHideTokenSections: Int {
     }
 }
 
-struct AddHideTokensViewModel {
+class AddHideTokensViewModel {
```

Comment:
Why do you change it from a struct to a class?

Reply:
If there's a special reason, can you leave a comment like in `TokensViewModel`? Otherwise, it's easy in the future to change it back and the code only breaks silently.

Reply:
i changed it to class because we need to refresh view model in apply updates in only one place, then to not to update separately coordinator and view controller.

Reply:
ok, np.